interlude/src/Features/Score/Graph.fs

```diff
@@ -388,13 +407,13 @@ and ScoreGraph(score_info: ScoreInfo, stats: ScoreScreenStats ref) =
 
             let box =
                 Rect.Box(
-                    this.Bounds.Left + BOX_PADDING_X + percent * (this.Bounds.Width - BOX_WIDTH - BOX_PADDING_X - BOX_PADDING_X),
-                    this.Bounds.Top + box_padding_y,
+                    this.Bounds.Left + percent * (this.Bounds.Width - BOX_WIDTH),
+                    this.Bounds.Top - 250f,
                     BOX_WIDTH,
                     this.Bounds.Height - box_padding_y - box_padding_y
```

Comment:
```suggestion
                    BOX_HEIGHT
```
`this.Bounds.Height - box_padding_y - box_padding_y` = `BOX_HEIGHT` so this line is simpler and equivalent

I wrote it that way because it acted as a mental note to me that the box was vertically centred in the graph, but it definitely doesn't make sense now (and probably didn't even before), since the box is now above the graph

This also means the `box_padding_y` variable can be deleted too